queries/tiger/highlights.scm

```diff
@@ -0,0 +1,125 @@
+; Built-ins {{{
+((function_call
+  function: (identifier) @function.builtin)
+  (#match? @function.builtin "^(chr|concat|exit|flush|getchar|not|ord|print|print_err|print_int|size|strcmp|streq|substring)$")
+  ; FIXME: not supported by neovim
+  ; (#is-not? local)
```

Comment:
it actually is, but very slow. you would have to use the names from `locals.scm`. so `(#not-is @function.builtin var)`

Reply:
Aaah, that makes a lot of sense given the `#set!` for different kinds of locals.

Once again a place where Neovim diverges from the `tree-sitter` CLI it seems. As a new contributor I think it would be good to document those :-)

Reply:
> it actually is, but very slow.

So the general advice would be to avoid using that feature I suppose?

Reply:
They were once documented, but we removed all usages and docs because they were to slow. The querying of the locals uses an in efficient algorithm for the query results and for resolving the locals